padraic/file_get_contents/CVE-2016-5385.yaml

```diff
@@ -0,0 +1,7 @@
+title:     HTTP Proxy header vulnerability
+link:      https://github.com/humbug/file_get_contents/releases/tag/1.1.2
+cve:       CVE-2016-5385
+branches:
```

Comment:
The `branches` entry seems invalid here, as we are missing a branch name reference

Reply:
indeed :(

Reply:
I'll send a patch in a few moments :-)

Reply:
https://github.com/FriendsOfPHP/security-advisories/pull/252

Reply:
Ah damn, sorry about that looks like I deleted the line by accident. Thanks Marco for the patch